### ISSUE TITLE
Reading Time

### ISSUE THREAD

**Issue by duhanmeric** [scrut-ch/issues#3]
It would be very good to add "reading time" as a feature. I believe it would be useful for us to directly understand how long the text we are writing is, even if it works with a very simple algorithm.

**Comment by intellent** [comment]
Good idea. Shouldn’t be too hard to implement. The tricky part is to figure out where to place this additional info.
